sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,142 @@
+  - Made host device return `nullptr` for bad USM allocation functions
+    (huge, 0, etc) [2a000d9]
+  - Added templated forms of USM allocation functions [42cf5bf]
+  - Added support for APIs that query properties of USM pointers [926e38e]
+  - Added clean up finished command nodes of the execution graph in the
+    situations when the wait for a command to complete is called implicitly
+    or explicitly [438dc49]
+  - Added 2 `sycl::queue` constructors accepting `sycl::context` and
+    `sycl::device` arguments [c81c1c5]
+
+### Documentation
+  - Added [documentation](doc/extensions/QueueShortcuts/QueueShortcuts.adoc) for
+    simplification of the `sycl::queue` functions [478b7c0]
+  - Added [documentation](doc/extensions/ReqdWorkGroupSize/SYCL_INTEL_reqd_work_group_size.asciidoc)
+    for `reqd_work_group_size` extension [c2c416a]
+  - The FAQ [document](doc/FAQ.md) was introduced [e42b40e]
+  - Added Ordered Queue Property
+    [proposal](doc/extensions/OrderedQueue/OrderedQueue_v2.adoc) [9fa878f]
+  - Added device code split options documentation to the
+    [user's manual](doc/SYCLCompilerUserManual.md) [1355aa6]
+  - Added documentation for [ExtendedAtomics extension](doc/extensions/ExtendedAtomics/SYCL_INTEL_extended_atomics.asciidoc) [4445462]
+  - Removed old Ordered Queue proposal and make a note of deprecation [e8f1f29]
+
+## Bug fixes
+### SYCL Frontend and driver changes
+  - Fixed variable sharing passed by value to `parallel_for_work_group`
+    [d8ea63a]
+  - Do not produce an error if some restricted feature (e.g. exceptions) is used
+    by a function only referenced in unevaluated contexts [5eae571]
+  - Fixed problem with not cleaning up temporary files when device code split
+    feature is enabled [d86ee2f]
+  - Fixed issue with emitting wrong 'typename' keywords in the integration
+    header [c19372e]
+  - The SYCL target image registration functions have been renamed to avoid
+    conflicts with the OpenMP registration functions [82fd970]
+  - Avoid using `std::memcpy` in the device code [f39f47e]
+  - Fixed `-save-temps` option when used along with `-fsycl` [f7f4699]
+  - Also fixed link steps triggering for libraries specified using
+    `-foffload-static-lib` when no source or object file is provided [360b25b]
+
+### SYCL headers and runtime
+  - Fixed final result saturation in the host implementation of `sycl::mad_sat`
+    [54dddb4]
+  - Fixed a crash when a non-nullptr_t `0x0` value is passed to the
+    `sycl::buffer::set_final_data` method [6a0e279]
+  - Fixed an issue with copying sub-buffer between different contexts [0867a38]
+  - Resolved the problem when local accessor is a temporary object [1eed329]
+  - Fixed an issue with event not being retained when memory object is
+    constructed using interoperability constructors [0aabe7e]
+  - Fixed compilation of kernels which use `sycl::stream` for FPGA device
+    [c4dbaa2]
+  - Fixed execution graph cleanup on memory object destruction [7a75b54]
```

Comment:
Should this be merged with the original feature description?

Reply:
I don't think so, this fix is for a different cleanup mechanism that existed before.